packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -0,0 +1,29 @@
+import { UserAgentApplication as AzureActiveDirectory } from 'msal'
+
+export type { AzureActiveDirectory }
+import type { AuthClient } from './'
+
+export type AzureActiveDirectoryClient = AzureActiveDirectory
+export interface AzureActiveDirectoryUser {}
+
+export const azureActiveDirectory = (client: AzureActiveDirectoryClient): AuthClient => {
+  return {
+    type: 'azureActiveDirectory',
+    client,
+    restoreAuthState: async () => {},
```

Comment:
Do you need/want to specify this one?

Reply:
I can't find any related method in [MSAL's UserAgentApplication](https://pub.dev/documentation/msal_js/latest/msal_js/UserAgentApplication-class.html#constructors). Removed in https://github.com/jeliasson/redwood/commit/d458839a1606ed5b174393bc73595ad0a8dac438